### PR TITLE
Add success title slide-up to Loader

### PR DESCRIPTION
**Pull request by sisou**

### CODE REVIEW

File: src/components/Loader.vue [hunk 1]
```diff
@@ -395,4 +395,23 @@ export default Loader;
         max-width: 70%;
         opacity: 1;
     }
+
+    .loader.exit-transition {
+        /* animation: exit-transition 600ms 1s; */
```
Comment (Bettelstab) [inline src/components/Loader.vue]: delete this...

File: src/components/Loader.vue [hunk 2]
```diff
@@ -395,4 +395,23 @@ export default Loader;
...
+    }
+
+    @keyframes exit-transition {
+        from { transform: scale(1); opacity: 1; }
+        80%  { opacity: 0; }
+        to   { transform: scale(0); opacity: 0; }
+    }
```
Comment (Bettelstab) [inline src/components/Loader.vue]: ...until here, I guess?

File: src/components/Loader.vue [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <template>
-    <div class="loader" :class="showLoadingBackground && (lightBlue ? 'nq-light-blue-bg' : 'nq-blue-bg')">
+    <div class="loader" :class="[showLoadingBackground && (lightBlue ? 'nq-light-blue-bg' : 'nq-blue-bg'), state === 'success' && 'exit-transition']">
```
Comment (danimoh) [inline src/components/Loader.vue]: Would be nicer to use
```suggestion
    <div class="loader" :class="{ 'nq-blue-bq': showLoadingBackground && !lightBlue, 'nq-light-blue-bg': showLoadingBackground && lightBlue, 'exit-transition': state === 'success' }">
```

File: src/components/Loader.vue [hunk 4]
```diff
@@ -395,4 +395,23 @@ export default Loader;
...
+        /* animation: exit-transition 600ms 1s; */
+    }
+
+    @keyframes exit-transition {
+        from { transform: scale(1); opacity: 1; }
+        80%  { opacity: 0; }
+        to   { transform: scale(0); opacity: 0; }
+    }
+
+    .loader.exit-transition .success .icon-row {
```
Comment (danimoh) [inline src/components/Loader.vue]: What's actually the reason for the unscoped second `<style>` tag and manual scoping instead?

File: src/components/Loader.vue (hunk 2 above)
Comment (danimoh) [reply src/components/Loader.vue]: The whole `exit-transition` class + logic can then also be removed, as currently it only affects `.success .icon-row` which is only visible on success anyways.

File: src/components/Loader.vue (hunk 4 above)
Comment (sisou) [reply src/components/Loader.vue]: Slots do not inherit scoped styles. Thus when we use custom HTML for slots from outside, those can only be styled with global styles.

**Pull request merged**